.pylintrc

```diff
@@ -32,7 +32,7 @@ load-plugins=
 #R0903 = Too few public methods (%s/%s)
 #C0326 = bad-whitespace
 #I0011 = locally-disabling
-disable=R0903,C0326,I0011,redefined-variable-type,c-extension-no-member
+disable=R0903,C0326,I0011,redefined-variable-type,c-extension-no-member,W,C,R
```

Comment:
What are these for?

Reply:
It allows people to run `pylint src` instead of using the runPyLint.sh script. See also: https://github.com/DIRACGrid/DIRAC/pull/4990#discussion_r582130798

Reply:
I don't think all of this should be disabled (see https://github.com/DIRACGrid/DIRAC/pull/4990#discussion_r582607709)